oci/layout/oci_dest_test.go

```diff
@@ -58,3 +58,19 @@ func TestPutBlobDigestFailure(t *testing.T) {
+
+	err = ociDest.PutManifest([]byte(m))
+	require.Error(t, err)
+	assert.Contains(t, `Unrecognized manifest media type: "application/vnd.docker.distribution.manifest.v1+prettyjws"`, err.Error())
```

Comment:
> this test will permanently break…

yep, that'll be the time we'll update it I guess but right now this is tracking a specific bug